docs/TROUBLE.md

```diff
@@ -0,0 +1,162 @@
+
+- For CRSF, check that you have correctly [configured ArduPilot](ARDUPILOT.md#crsf-receiver) to use this input.
+
+- For CRSF, check that you have correctly [configured your Rx](CRSF.md#mlrs-rx-module-setup) for crsf output.
+
+- If your Rx [doesn't provide an RC output](ELRS_RECEIVERS.md#connections) or you don't want to use it, check that you have set [Rx Snd RcChannel](PARAMETERS.md#rx-snd-rcchannel) to rc override.
+
+## 9. Confirm reliable message delivery
+
+Use your ground control station connected to your Tx module to examine the message loss rate both during parameter download and for a few minutes after it completes.
+
+For QGroundControl, select Application Settings -> MAVLink and scroll down to MAVLink Link Status.
+
+For Mission Planner, select status in the upper right, move the windows so you can see both and then refresh parameters from the full parameters page.
+
+You should see very few or no messages lost.
+
+- If you see more than the occasional lost message or if parameter download takes more time than expected (8 to 60 seconds, depending on mode and ArduPilot version), check that [rx-ser-link-mode](PARAMETERS.md#rx-ser-link-mode) is set to "mavlink" or "mavlinkX" and that [rx-snd-radiostat](PARAMETERS.md#rx-snd-radiostat) is set to "ardu_1" or "meth_b"
+
+- Check your [ArduPilot setup](CRSF.md#ardupilot-setup) to be sure that the baud rate between the Rx and  and your configured Stream Rates are appropriate for your radio mode.
+
+- Upgrade to a newer version of ArduPilot which may have fixed flow control issues.
+
+## Range issues
+
+If you don't see the range you expected, consider the following.
+
+- Never power up a Rx or Tx module without an antenna connected as it may cause damage, especially at high output levels.
+
+- When choosing and connecting antennas be aware of the difference between SMA and RP-SMA.  It is unfortunately possible to attach an RP-SMA antenna to an SMA jack but there will be no electrical connection of the signal (see above).
+
+- If your model allows, always use a vertical antenna orientation on both the Tx module and the Rx.  Especially with non-diversity systems.
+
+- Be aware that calculated/estimated maximum ranges are free-space line of sight numbers and don't take into account obstructions or near by objects such as trees and buildings.  Nor do they take into account RF noise from electronics on the craft or other users of the band.
+
+- Not all antennas are created equal and some are surprisingly bad.
+
+- Directional antennas, especially on the Tx module can increase or decrease range depending on antenna orientation.
+
+- Metal, body parts, CF frames and wires near the antenna can cause reduced range.
+
+## Fail-safe testing
+
+It is critical to check and test your failsafe behavior and configuration *before* you experience a connection loss.
+
+- The "no signal" method is highly recommended.
+
+- See [rx-failsafe-mode](PARAMETERS.md#rx-failsafe-mode).
```

Comment:
no such parameter ... (resolution see before)

Reply:
what below?  Do you just mean I got the parameter name wrong? (case and dashes)

Reply:
"32 hidden conversations"  (Why didn't I notice that before)

Reply:
"... see the "Rx Failsafe Mode" parameter."